Wordle.Api/Wordle.Api/Controllers/WordDateController.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.AspNetCore.Mvc;
+using Wordle.Api.Models;
+using Wordle.Api.Services;
+using Wordle.Api.Dtos;
+
+namespace Wordle.Api.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class WordDateController(WordDateService wordDateService) : ControllerBase
```

Comment:
I really like how you have multiple controller methods for each function you need.

Reply:
Thank you!